nifi-nar-bundles/nifi-elasticsearch-5-bundle/nifi-elasticsearch-5-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,252 @@
+nifi-elasticsearch-5-nar
+Copyright 2015-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) Elasticsearch
```

Comment:
ES 5 brings in "io.netty:netty:jar:3.10.6.Final:compile" which is ASLv2 licensed but has the following NOTICE file which is much more than what you have listed below.

https://github.com/netty/netty/blob/netty-3.10.6.Final/NOTICE.txt